README.md

```diff
@@ -30,40 +31,36 @@ Currently it shows:
   * `⇡` — ahead of remote branch;
   * `⇣` — behind of remote branch;
   * `⇕` — diverged chages.
-* Mercurial repo status:
+* Current Mercurial branch and rich repo status:
   * `?` — untracked changes;
   * `+` — uncommitted changes in the index;
   * `!` — unstaged changes;
   * `✘` — deleted files;
-* Prompt character turns red if the last command exits with non-zero code.
```

Comment:
This is still true, keep it maybe?

Reply:
It's now the second item of the list. Look up ☝️

Reply:
@maximbaz please, click **Resolve** button to hide this thread.

Reply:
I don't have it...?

![image](https://user-images.githubusercontent.com/1177900/34958879-19791be4-fa34-11e7-964c-1b6b36cb1e75.png)